kafka-merge-pr.py

```diff
@@ -70,7 +70,7 @@
 
 DEV_BRANCH_NAME = "trunk"
 
-DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "2.3.2")
+DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "2.3.3")
```

Comment:
Same as above.

Reply:
This was already set to `2.3.2` though - checking https://kafka.apache.org/downloads though, the last AK release for this line was `2.3.1`, so I assume the next one is `2.3.2`.